includes/API/ApiLabkiUpdate.php

```diff
@@ -178,12 +349,9 @@ public function getAllowedParams(): array {
             'finalTitle'    => [ self::PARAM_TYPE => 'string', self::PARAM_DFLT => null ],
             'pageNamespace' => [ self::PARAM_TYPE => 'integer', self::PARAM_DFLT => 0 ],
             'wikiPageId'    => [ self::PARAM_TYPE => 'integer', self::PARAM_DFLT => 0 ],
-            // removePage convenience
             'pageId'        => [ self::PARAM_TYPE => 'integer', self::PARAM_DFLT => null ],
-            // Bulk pages for install/update/remove verification
             'pages'         => [ self::PARAM_TYPE => 'string', self::PARAM_DFLT => null ],
+            'packs'         => [ self::PARAM_TYPE => 'string', self::PARAM_DFLT => null ],
```

Comment:
looks like we now have two calling conventions for this endpoint: looks like there is an old version for a single pack and this new version for multiple packs. probably want to only do one